include/souper/Infer/AbstractInterpreter.h

```diff
@@ -201,12 +201,20 @@ namespace souper {
     ForcedValueAnalysis(Inst *RHS_) : RHS(RHS_), Conflict(false) {
       countSymbolicInsts(RHS);
     }
-    struct Value {
-      llvm::APInt Concrete;
-      // Add Range and Knownbits, maybe
+    class Value {
+    public:
+      Value(llvm::APInt Val_) : Val{Val_} {}
       bool conflict(Value &Other) {
-        return Concrete != Other.Concrete;
+        if (hasConcrete() && Concrete() != Other.Concrete()) {
```

Comment:
don't write code of the form

if (x) return true; else return false;

go back to the previous syntax, and just change the condition